nservicebus/messaging/transactions.md

```diff
@@ -119,6 +119,7 @@ WARNING: This might escalate to a distributed transaction if data in different d
 
 The following options for transaction scopes used during message processing can be configured.
 
+NOTE: In Version 6 isolation level and timeout for transaction scopes are also configured at the transport level.
```

Comment:
This is a fragile statement. What about version 5 and lower? When we release v7 does this apply? You seem to imply it is only v6. Perhaps "in version 6 and higher..."